### PR TITLE
Add security model

### PR DESCRIPTION
**Pull request by chaen**
Green light from Mischa Salle

image

### CODE REVIEW

**Comment by chaen** [comment]
Hum, I was expecting the TOC to work :thinking: 
https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

**Comment by chrisburr** [comment]
> Hum, I was expecting the TOC to work

That refers to this button:

<details><summary>Details</summary>

Screenshot 2023-10-12 at 06 34 30

</details>

File: security_model.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+
+In the event of malicious activity, refer to existing procedures in addition to what was previously described.
+
+## Changelog
+
+* v0.1.0 (2023-06-07): Initial draft for internal review.
+* v0.2.0 (2023-06-12): Add glossary of common terms.
+* v0.3.0 (2023-06-27):
+    * Define Identity
+    * Specify tokem issuer
```
Comment (aldbr) [inline security_model.md:275]:
```suggestion
    * Specify token issuer
```

File: security_model.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+* **Authorization Code with Proof Key of Code Exchange:** This is primarily used for the DiracX web portal.
+* **Device Authorization Flow with Proof Key of Code Exchange:** This is primarily used for terminal access to DiracX services.
+* **Refresh Token Grant:** This is used with the refresh tokens issued with the other flows.
+* **Token Exchange Grant:** Enables trusted externally issued tokens to be exchanged for DiracX credentials.
+
+There is currently no need foreseen for DiracX to issue `id_tokens` as all the information is already contained in the access token.
+
+All flows involving the external IdP follow the same sequence:
+
+![](https://codimd.web.cern.ch/uploads/upload_28fc4b6fce8ed1548f9014fd956d16af.png)
```
Comment (aldbr) [inline security_model.md:138]: I guess you need to add the image to the PR

**Pull request merged**